sdk/monitor/opentelemetry-exporter/package.json

```diff
@@ -41,17 +40,17 @@
   "devDependencies": {
     "@types/mocha": "^7.0.2",
     "@types/node": "^14.0.23",
-    "@typescript-eslint/parser": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^3.0.0",
+    "@typescript-eslint/parser": "^2.0.0",
+    "@typescript-eslint/eslint-plugin": "^2.0.0",
     "c8": "^7.2.0",
-    "eslint": "^7.0.0",
+    "eslint": "^6.1.0",
     "eslint-config-prettier": "^6.0.0",
     "eslint-plugin-import": "^2.21.2",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.4",
     "mocha": "^7.1.1",
-    "nock": "^13.0.0",
-    "prettier": "^2.0.0",
+    "nock": "^12.0.3",
+    "prettier": "^1.16.4",
```

Comment:
I feel bad seeing you go backwards here. Maybe we can upgrade these everywhere soon